src/manifest.cpp

```diff
@@ -392,8 +392,8 @@ verify_result(const Context& ctx,
               std::unordered_map<std::string, FileStats>& stated_files,
               std::unordered_map<std::string, digest>& hashed_files)
 {
-  for (uint32_t i = 0; i < result.file_info_indexes.size(); ++i) {
-    const auto& fi = mf.file_infos[result.file_info_indexes[i]];
+  for (unsigned int file_info_indexe : result.file_info_indexes) {
```

Comment:
`indexe` -> `index`
`unsigned int` -> `uint32_t`

Reply:
I'll attempt to fix that second issue in clang-tidy or file a report otherwise.